### ISSUE TITLE
Implement caching to reduce number of calls to the API endpoint

### ISSUE THREAD

**Issue by morozgrafix** [morozgrafix/MMM-horoscope#1]
Since most of the time horoscopes change only once a day there is no need to hammer API with requests. I wonder if it would be possible to implement some sort of caching mechanism to reduce number of calls to the API endpoint.

One potential solution is to use [NodeCache](https://www.npmjs.com/package/node-cache) package

**Comment by cowboysdude** [comment]
Or just set your update interval higher and have it check less frequently that way...

**Comment by morozgrafix** [comment]
Yes absolutely. I think it was more of a technical challenge to myself.
